Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -235,6 +235,32 @@ public override async Task<bool> RequestAccessTokenAsync(CancellationToken taskC
             return await base.GetAccessTokenForRequestAsync(authUri, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <inheritdoc/>
+        public Task<OIdCToken> GetOIdCTokenAsync(OIdCTokenOptions options, CancellationToken cancellationToken = default)
+        {
+            options.ThrowIfNull(nameof(options));
+            // If at some point some properties are added to OIdCToken that depend on the token having been fetched
+            // then initialize the token here.
+            TokenRefreshManager tokenRefreshManager = null;
```

Comment:
Same comment as per ComputeCredential - caching would be nice.

Reply:
Yes, same here, caching would need to be done on a per options basis, happy to do that though.